iXBRLViewerPlugin/viewer/src/less/viewer.less

```diff
@@ -141,3 +141,23 @@ div.ixbrl-table-handle {
     }
   }
 }
+
+.review.review-highlight-untagged-numbers .review-untagged-number {
+  background-color: @highlight-1;
+}
+
+.review.review-highlight-untagged-dates .review-untagged-date {
+  background-color: @highlight-2;
+}
+
+.review .ixbrl-highlight:not(.ixbrl-no-highlight) {
+  background-color: @highlight-default !important;
```

Comment:
Now that this is all in one file, it'd be nice to be able to lose the `!important`s on these lines, which I think we could do by adding a `:not(.review)` block where the highlighting is originally applied around line 11, and then dropping these overrides altogether.

Reply:
Actually I think because `.review .ixbrl-highlight:not(.ixbrl-no-highlight)` is narrower than `.ixbrl-highlight:not(.ixbrl-no-highlight)` it automatically supersedes the default. Though I'm not 100% confident in my understanding, it appears to have the desired end result when I simply exclude `!important`.

Reply:
Unfortunately this doesn't work for values within tables without `!important`. The alternate table row highlighting is applied with inline styles and takes a higher precedence

Reply:
Yes, sorry, I was misremembering what this `!important` was for.  As you say, it's to override explicit background colors in the source document, rather than to override base styles from the plugin.

I think it would make sense to re-order the file so that these styles are immediately after the non-review styles.

Reply:
It would also be good to add a comment explaining why the `!important`s are there